smallslive/templates/account/event-support.html

```diff
@@ -139,9 +150,12 @@
     <button id="confirmButton" data-hide='true' class="button-bordered">Confirm</button>
   {% else %}
     <a href="{% url 'event_detail' event_id event_slug %}">
-      <button>Continue watching</button>
+      <button>Return to Event</button>
     </a>
   {% endif %}
 {% endblock %}
+<<<<<<< HEAD
```

Comment:
@Aslan1st is this correct?

Reply:
@mprunell no! it slipped in when I fixed a conflict I'm editing now